code/_helpers/global_lists.dm

```diff
@@ -56,32 +56,8 @@ var/global/list/facial_hair_styles_female_list = list()
 var/global/list/skin_styles_female_list = list()		//unused
 var/global/list/body_marking_styles_list = list()
 var/global/list/chargen_disabilities_list = list()
-	//Underwear
-var/global/list/underwear_m = list("White" = "m1", "Grey" = "m2", "Green" = "m3", "Blue" = "m4", "Black" = "m5", "Mankini" = "m6", "Boxers" = "boxers", "Green and blue boxers" = "boxers_green_and_blue","Loveheart boxers" = "boxers_loveheart","None") //Curse whoever made male/female underwear diffrent colours
-var/global/list/underwear_f = list("Red" = "f1", "White" = "f2", "Yellow" = "f3", "Blue" = "f4", "Black" = "f5", "Thong" = "f6", "Black Sports" = "f7","White Sports" = "f8","None")
-	//undershirt
-var/global/list/undershirt_t = list(
-	"White tank top" = "u1", "Black tank top" = "u2", "Black shirt" = "u3", "White shirt" = "u4",
-	"Polo" = "polo", "Blue polo" = "bluepolo_s", "Red polo" = "redpolo_s", "Yellow polo" = "grayyellowpolo_s",
-	"Striped shirt" = "shirt_stripes_s", "NanoTrasen shirt" = "shirt_nano_s", "Tiedye shirt" = "shirt_tiedye_s",
-	"Green sport shirt" = "greenshirtsport_s", "Red sport shirt" = "redshirtsport_s", "Blue sport shirt" = "blueshirtsport_s",
-	"Striped tank top " = "tank_stripes_s", "Rainbow tank top" = "tank_rainbow_s", "Longsleeve shirt" = "undershirt_long",
-	"Striped longsleve shirt" = "longstripe", "Blue striped longsleve shirt" = "longstripe_blue", "None")
-
-	//socks
-var/global/list/socks_f = list(
-	"White normal" = "white_norm", "White short" = "white_short", "White knee" = "white_knee",
-	"White thigh" = "white_thigh", "Black normal" = "black_norm", "Black short" = "black_short",
-	"Black knee" = "black_knee", "Black thigh" = "black_thigh", "Striped normal" = "striped_norm",
-	"Striped short" = "striped_short", "Striped knee" = "striped_knee", "Striped thigh" = "striped_thigh",
-	"Rainbow normal" = "rainbow_norm", "Rainbow short" = "rainbow_short", "Rainbow knee" = "rainbow_knee",
-	"Rainbow thigh" = "rainbow_thigh", "Pantyhose" = "pantyhose", "None")
-
-var/global/list/socks_m = list(
-	"White normal" = "white_norm", "White short" = "white_short", "White knee" = "white_knee",
-	"Black normal" = "black_norm", "Black short" = "black_short", "Black knee" = "black_knee",
-	"Striped normal" = "striped_norm", "Striped short" = "striped_short", "Striped knee" = "striped_knee",
-	"Rainbow normal" = "rainbow_norm", "Rainbow short" = "rainbow_short", "Rainbow knee" = "rainbow_knee", "None")
+
+var/datum/category_collection/underwear/global_underwear = new()

```

Comment:
This is technically not a list. 🤔 And should probably be in the global objects file. Tho I'm not too sure I like that file in the first place. HMM.

Reply:
🤔🤔🤔🤔🤔🤔🤔

Reply:
<details>
<summary>🤔 Arc</summary>
<img src="https://user-images.githubusercontent.com/3950115/66249965-a36ed500-e6f0-11e9-9e83-8cf8fd3af44f.png">
</details>